vendor/github.com/Azure/azure-docker-extension/pkg/vmextension/handlerenv.go

```diff
@@ -3,8 +3,18 @@ package vmextension
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
 )
 
+// HandlerEnvFileName is the file name of the Handler Environment as placed by the
```

Comment:
meh, both are the same in this case. also if we need to fix it, it'd be better to fix it upstream